contrib/hack/coverage.sh

```diff
@@ -18,7 +18,7 @@ set -o nounset
 set -o errexit
 
 ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
```

Comment:
The intent with `ROOT` is to be the main `kubernetes-incubator/service-catalog` directory, so instead of changing L21, I'd suggest adding another `/..` to the path here.

Reply:
Good point! Thanks!